recipes-debian/postfix/postfix_debian.bb

```diff
@@ -0,0 +1,307 @@
+#base recipe: meta-openembedded/meta-networking/recipes-daemons/postfix/postfix_3.4.23.bb
+#base branch: hardknott
+#base commit: 786d3aa313e31a718acc9611ba3adb60573ec458
+
+SUMMARY = "Postfix Mail Transport Agent"
+DESCRIPTION = "Postfix is Wietse Venema's mail server that started life at \
+IBM research as an alternative to the widely-used Sendmail program. \
+Postfix attempts to be fast, easy to administer, and secure. The outsidei \
+has a definite Sendmail-ish flavor, but the inside is completely different."
+
+HOMEPAGE= "http://www.postfix.org"
+SECTION = "mail"
+
+DEPENDS = "db icu libpcre libnsl2 openssl postfix-native \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'ldap', 'openldap', '', d)} \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'sasl', 'cyrus-sasl', '', d)} \
+"
+
+LICENSE = "IPL-1.0 | EPL-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=b181651ad99a7dc4cc8c4ce2f491ed1a"
+
+S = "${WORKDIR}/postfix-${PV}"
+
+CLEANBROKEN = "1"
+
+BBCLASSEXTEND = "native"
+
+inherit update-rc.d useradd update-alternatives systemd lib_package
+
+inherit debian-package
+require recipes-debian/sources/postfix.inc
+
+INITSCRIPT_NAME = "postfix"
+INITSCRIPT_PARAMS = "start 58 3 4 5 . stop 13 0 1 6 ."
+USERADD_PACKAGES = "${PN}-bin"
+USERADD_PARAM_${PN}-bin = \
+"-d /var/spool/postfix -r -g postfix --shell /bin/false postfix; \
+ -d /var/spool/vmail -r -g vmail --shell /bin/false vmail \
+"
+GROUPADD_PARAM_${PN}-bin = "--system postfix;--system postdrop;--system vmail"
+
+export SYSLIBS = "${LDFLAGS}"
+
+# CCARGS specifies includes, defines
+# AUXLIBS specifies libraries
+# Linux2/Linux3 has HAS_DB defined
+# makedefs will make CC to be CC+CCARGS
+
+# ldap support
+export CCARGS-ldap  = "\
+    ${@bb.utils.contains('DISTRO_FEATURES', 'ldap', '-DHAS_LDAP', '', d)}"
+export AUXLIBS-ldap = "\
+    ${@bb.utils.contains('DISTRO_FEATURES', 'ldap', '-lldap -llber', '', d)}"
+
+# no native openldap
+export CCARGS-ldap_class-native = ""
+export AUXLIBS-ldap_class-native = ""
+
+export CCARGS-nonis_libc-musl = "-DNO_NIS"
+export CCARGS-nonis = ""
+
+# SASL support -DUSE_LDAP_SASL -DUSE_SASL_AUTH
+# current openldap didn't enable SASL
+export CCARGS-sasl  = "\
+    ${@bb.utils.contains('DISTRO_FEATURES', 'sasl', '-DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I${STAGING_INCDIR}/sasl', '', d)}"
+export AUXLIBS-sasl = "\
+    ${@bb.utils.contains('DISTRO_FEATURES', 'sasl', '-lsasl2', '', d)}"
+export CCARGS-sasl_class-native = ""
+export AUXLIBS-sasl_class-native = ""
+
+# PCRE, TLS support default
+export CCARGS  = "${CFLAGS} -DHAS_PCRE -DUSE_TLS -I${STAGING_INCDIR}/openssl ${CCARGS-ldap} ${CCARGS-sasl} ${CCARGS-nonis}"
+export AUXLIBS = "-lpcre -lssl -lcrypto ${AUXLIBS-sasl} ${AUXLIBS-ldap}"
+export POSTCONF = "${STAGING_DIR_NATIVE}${sbindir_native}/postconf"
+
+# OPT,DEBUG is aready in CFLAGS
+# ignore the OPTS="CC=$CC" in Makefile it will not use the CC=$CC $CCARGS
+EXTRA_OEMAKE += "OPT= DEBUG= OPTS= "
+
+do_debian_unpack_extra_append () {
```

Comment:
@AokiNobuhiko-Metatechno  Why do you remove these patches?

Reply:
These patches modify the documentation path to .gz like below.
```
-$manpage_directory/man1/mailq.1:f:root:-:644
+$manpage_directory/man1/mailq.1.gz:f:root:-:644
```

However, when I apply this patch and build, I get the error like below during do_install task.
```
| Updating XXX/build/tmp-glibc/work/x86_64-linux/postfix-native/3.4.23-r0/image/XXX/build/tmp-glibc/work/x86_64-linux/postfix-native/3.4.23-r0/recipe-sysroot-native/usr/share/man/man1/mailq.1.gz...
| cp: cannot stat 'man/man1/mailq.1.gz': No such file or directory
```

Documentation included in other packages is not .gz.
```
$ oe-pkgdata-util list-pkg-files bash-doc
bash-doc:
	/usr/share/doc/bash/CHANGES
	/usr/share/doc/bash/COMPAT
	/usr/share/doc/bash/INTRO
	/usr/share/doc/bash/NEWS
	/usr/share/doc/bash/POSIX
	/usr/share/doc/bash/RBASH
	/usr/share/doc/bash/README
	/usr/share/doc/bash/bash.html
	/usr/share/doc/bash/bashref.html
	/usr/share/info/bash.info
	/usr/share/man/man1/bash.1
	/usr/share/man/man1/bashbug.1
```

Therefore,  I removed this patches so as not to add .gz to the document's path.

Reply:
I got it. thanks.
please write such description when you needed to change like that in a recipe or commit log.

Reply:
ah, you don't have to update recipe this time because you already described that in this pr.